tests/test_board.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from chess import pieces
+from chess.board import Board, check_position_format
+from chess.game import Game
+
+
+def test_check_position_format():
+    with pytest.raises(ValueError):
+        check_position_format("A77")
+    with pytest.raises(ValueError):
+        check_position_format("A9")
+    with pytest.raises(ValueError):
+        check_position_format("77")
+
+
+def test_board_getitem():
```

Comment:
Added tests for each function. If a function is untested, you should assume it does not work